Soccar [Game-Server]/src/nl/soccar/gameserver/model/session/RoomWrapper.java

```diff
@@ -34,11 +36,23 @@
     private final SessionWrapper session;
     private final Room room;
 
+    /**
+     * Intializes the RoomWrapper class.
+     *
+     * @param session The given session, not null.
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)